interpreter/k8s/src/main/scala/io/buoyant/transformer/k8s/LocalNodeTransformerInitializer.scala

```diff
@@ -0,0 +1,20 @@
+package io.buoyant.transformer.k8s
+
+import io.buoyant.namer.{NameTreeTransformer, TransformerConfig, TransformerInitializer}
+import java.net.InetAddress
+
+class LocalNodeTransformerInitializer extends TransformerInitializer {
+  val configClass = classOf[LocalNodeTransformerConfig]
+  override val configId = "io.l5d.localnode"
+}
+
+class LocalNodeTransformerConfig extends TransformerConfig {
+
+  override def mk: NameTreeTransformer = {
```

Comment:
i prefer parens for `mk()` since it reads imperative and not something like `toTransformer`  (also, from the interface point of view, it may be side-effecty).